sanji/core.py

```diff
@@ -134,8 +135,11 @@ def ___dispatch(handler, message):
 
         try:
             for result in results:  # same route
-                map(lambda handler: ___dispatch(handler, result["message"]),
+                data = map(lambda handler: ___dispatch(
```

Comment:
why we need to get `data` and `list` it?

Reply:
Map in Python2 will return a `list object` but `map object` in Python3. It causes the lambda should execute the handler to do the dispatch but nothing happened. Here I add a workaround which casts the map object to list, then the result becomes what we expect.